install_config/web_console_customization.adoc

```diff
@@ -376,6 +376,41 @@ assetConfig:
 ----
 ====
 
+. As a cluster administrator, run the following command to give users the
```

Comment:
this is really specific to jenkins pipelines, this wont be true of all tech preview features